dev/app/app.js

```diff
@@ -4,16 +4,12 @@ var angular = require('angular');
 require('angular-route');
 require('angular-sanitize');
 require('angularfire');
-/**
- * @ngdoc overview
- * @name freeCodeLondonersCampApp
- * @description
- * # freeCodeLondonersCampApp
- *
- * Main module of the application.
- */
 
-var dataSource = (require('./app.config')) ? require('./shared/services/local') : require('./shared/services/firebase');
+require('./assets/sass/main.scss');
```

Comment:
we are not using React yet? are we?
it doesn't create probs but it doesn't seem right to have this here - I ll see if I can come out with better solution